.github/workflows/publish_artifacts.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+# Copyright (c) 2025 Qualcomm Innovation Center, Inc. All rights reserved.
+# SPDX-License-Identifier: MIT
```

Comment:
I guess the workflow files should have a license, the debos recipes and shell scripts use BSD-3-Clause and that's the license in the repo and usually what Qualcomm defaults to, so perhaps let's use BSD-3-Clause or we need to add MIT license to the repo as LICENSE.MIT or something.